todo_app/app.py

```diff
@@ -8,4 +9,23 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    items = sorted(items, key=lambda x: x["status"], reverse=True)
+    return render_template("index.html", items=items)
+
+@app.route('/add-item', methods=['POST'])
+def addItem():
+    add_item(request.form.get("newItem"))
+    return redirect(url_for('index')) 
```

Comment:
_Very_ minor & personal preference over real suggestion, but one I was talking with someone recently about was whether we preferred this way of finding address from function (`redirect(url_for('index'))`) or just the address itself (`redirect('/')`) and my instinct it for the latter - I think I'm more likely to be caught out by changing a function name and forgetting to update `url_for` than changing a route; probably just because that latter strikes me as more inherently risky so I'm more likely to search thoroughly.